src/js/components/agencyV2/visualizations/totalObligationsOverTime/AgencyBudgetLine.jsx

```diff
@@ -54,7 +54,7 @@ const AgencyBudgetLine = ({
                 }
             );
         }
-    }, [xScale, yScale, showTodayLineAndText]);
+    }, [xScale, yScale, showTodayLineAndText, agencyBudget, height, padding.bottom, padding.left, todaysDate, width]);
```

Comment:
Was this intentional or automatically added by a linter? I've had these auto-fixes really mess things up...

Reply:
This was the linter but I reviewed it. It make sense to add and doesn't appear to mess anything up.